virttest/utils_hotplug.py

```diff
@@ -483,3 +485,41 @@ def vcpuhotunplug_unsupport_str():
         return "not currently supported"
     else:
         return ""
+
+
+def create_mem_xml(params):
+    """
+    Create memory device xml.
+    """
+    mem_xml = memory.Memory()
+    mem_model = params.get("mem_model", "dimm")
+    mem_xml.mem_model = mem_model
+    tg_size = params.get("tg_size")
+    tg_sizeunit = params.get("tg_sizeunit", 'KiB')
+    tg_node = params.get("tg_node", 0)
+    pg_size = params.get("page_size")
+    pg_unit = params.get("page_unit", "KiB")
+    node_mask = params.get("node_mask", "0")
+    mem_addr = ast.literal_eval(params.get("memory_addr", "{}"))
+    numa_cells = params.get("numa_cells", "").split()
```

Comment:
most of the variables are not used more than once. so they can removed.

Reply:
yes, Currently it is used only once. Any further enhancements can use the variable. The one more reason for having variables to have all params.get grouped in one place. So that it will be easy for the user to find what what are the variable to be used in config file to use the API. It also give better readability.